streams/ir.m3u

```diff
@@ -71,3 +45,107 @@ https://hls.nejat.live/hls/stream.m3u8
+#EXTINF:-1 tvg-id="SL2.ir",SL 2 (720p)
+https://sltwohls.wns.live/hls/stream.m3u8
+#EXTINF:-1 tvg-id="IRIB1.ir",IRIB1 (1080p)
+https://ncdn.telewebion.com/tv1/live/playlist.m3u8
+#EXTINF:-1 tvg-id="IRIB2.ir",IRIB2 (1080p)
+https://ncdn.telewebion.com/tv2/live/playlist.m3u8
+#EXTINF:-1 tvg-id="IRIB3.ir",IRIB3 (Server 1)
+https://lenz.splus.ir/PLTV/88888888/224/3221226868/1.m3u8
+#EXTINF:-1 tvg-id="IRIB3.ir",IRIB3 (1080p)
+https://ncdn.telewebion.com/tv3/live/playlist.m3u8
+#EXTINF:-1 tvg-id="IRIB4.ir",IRIB4 (1080p)
+https://ncdn.telewebion.com/tv4/live/playlist.m3u8
+#EXTINF:-1 tvg-id="IRIB5.ir",IRIB5 (576p)
+https://ncdn.telewebion.com/tehran/live/playlist.m3u8
+#EXTINF:-1 tvg-id="Nasim.ir",IRIB Nasim (1080p)
+https://ncdn.telewebion.com/nasim/live/playlist.m3u8
+#EXTINF:-1 tvg-id="IRINN.ir",IRINN (1080p)
+https://ncdn.telewebion.com/irinn/live/playlist.m3u8
+#EXTINF:-1 tvg-id="IRINN2.ir",IRINN2 (576p)
+https://ncdn.telewebion.com/irinn2/live/playlist.m3u8
+#EXTINF:-1 tvg-id="VarzeshTV.ir",IRIB Varzesh (1080p)
+https://ncdn.telewebion.com/varzesh/live/playlist.m3u8
+#EXTINF:-1 tvg-id="Tamasha.ir",IRIB Tamasha (1080p)
+https://ncdn.telewebion.com/hdtest/live/playlist.m3u8
+#EXTINF:-1 tvg-id="PooyaTV.ir",IRIB Pooya & Nahal (1080p)
+https://ncdn.telewebion.com/pooya/live/playlist.m3u8
+#EXTINF:-1 tvg-id="iFilmPersian.ir",iFilm Persian (1080p)
+https://ncdn.telewebion.com/ifilm/live/playlist.m3u8
+#EXTINF:-1 tvg-id="NamayeshTV.ir",IRIB Namayesh (576p)
+https://ncdn.telewebion.com/namayesh/live/playlist.m3u8
+#EXTINF:-1 tvg-id="IRIBMostanad.ir",IRIB Mostanad (1080p)
+https://ncdn.telewebion.com/mostanad/live/playlist.m3u8
+#EXTINF:-1 tvg-id="QuranTV.ir",Iran Quran TV (1080p)
+https://ncdn.telewebion.com/quran/live/playlist.m3u8
+#EXTINF:-1 tvg-id="AmouzeshTV.ir",IRIB Amouzesh (576p)
+https://ncdn.telewebion.com/amouzesh/live/playlist.m3u8
+#EXTINF:-1 tvg-id="SalamatTV.ir",IRIB Salamat (576p)
+https://ncdn.telewebion.com/salamat/live/playlist.m3u8
+#EXTINF:-1 tvg-id="OfoghTV.ir",OfoghTV (576p)
+https://ncdn.telewebion.com/ofogh/live/playlist.m3u8
+#EXTINF:-1 tvg-id="IRIBOmid.ir",Omid (1080p)
+https://ncdn.telewebion.com/omid/live/playlist.m3u8
+#EXTINF:-1 tvg-id="Sepehr.ir",Sepehr (720p)
+https://ncdn.telewebion.com/sepehr/live/playlist.m3u8
+#EXTINF:-1 tvg-id="JameJamTVNetwork1.ir",JameJamTVNetwork1.ir (1080p)
+https://ncdn.telewebion.com/jjtv1/live/playlist.m3u8
+#EXTINF:-1 tvg-id="JameJamTVNetwork2.ir",JameJamTVNetwork2.ir (1080p) [Not 24/7]
+https://ncdn.telewebion.com/jjtv2/live/playlist.m3u8
+#EXTINF:-1 tvg-id="JameJamTVNetwork3.ir",JameJamTVNetwork3.ir (576p)
+https://ncdn.telewebion.com/jjtv3/live/playlist.m3u8
+#EXTINF:-1 tvg-id="AlWilayah.ir",Al Wilayah (720p) [Not 24/7]
+https://alwilayah.tv/cdn/alwilayah_tv-manifest.m3u8
+#EXTINF:-1 tvg-id="AsilTV.ir",Asil TV (720p)
+https://live.asil.tv/asiltv/index.m3u8
+#EXTINF:-1 tvg-id="",High Vision
+https://streamer1.connectto.com/HIGHVISION_WEB_1205/tracks-v1a1/mono.m3u8
```

Comment:
duplicate of the High Vision link below, right now it's having some severe difficulties